### ISSUE TITLE
Appendix G MaterialType doesn't exist in EN15907 

### ISSUE THREAD

**Issue by paulduchesne** [FIAF/fiaf-cataloguing-manual#31]
Appendix G MaterialType doesn't exist in EN15907 

Check and see whether it is called something else - it would be for designation of film/video/digital material types

**Comment by natashafairbairn** [comment]
I have corrected and updated contents now, and also double-checked all the other columns in Appendix G to make sure they are all correct.  

I hit one issue regarding bullet points not working though, e.g. see  first section of the table  - under Attributes of the entity and Attributes columns. I corrected the fact that the second row (with Serial and Collection listed) shouldn't be there and that everything in that row was actually a continuation of the first row. But the bullet points are not displaying as they should. The correct markdown hyphen is there in edit mode, but when put back into Preview mode it displays them runing together one after the other with a hyphen in between rather than creating the expected line break and bullet point mark.

Same with the Variant type row directly below it and with all other lists in the table. 

The line breaks also didn't work in some cases (but then did in others). I managed to put underlining back in though where it was supposed to occur.

Re-assigning to Paul for assistance with that issue.

**Comment by paulduchesne** [comment]
@paulduchesne to fix, the formatting from the original manual needs to be reflected in the markdown.

**Comment by heftberger** [comment]
Can we merge the branch to main and close this issue?

**Comment by natashafairbairn** [comment]
Not yet. Still needs some editing to get terms listing one under the other within the rows rather than side by side.

**Comment by heftberger** [comment]
Natasha wrote: Text was all sorted ages ago, there was just an outstanding issue with getting terms/rows to align properly – unless that has now been covered by this reference on the Issue? @paulduchesne

**Comment by paulduchesne** [comment]
No, I am still working through this, related to issue #12, which are both around our ability to configure tables properly.

**Comment by paulduchesne** [comment]
Fixed on 0eb8b71.
